frontends/web/src/routes/lightning/receive/receive.tsx

```diff
@@ -91,18 +95,28 @@ export function Receive() {
   }, [onPaymentsChange]);
 
   useEffect(() => {
-    setAmountSats(+amountSatsText);
-  }, [amountSatsText]);
+    setInputSats(+inputSatsText);
```

Comment:
please use `Number()` instead.. or maybe better change to inputSatsNumber using number and `target.valueAsNumber` since it is an input with type number.

Reply:
I left this comment and then realized that the state in this component should be simplified.. 
feel free to ignore.

Reply:
feel free to ignore expect for the type conversion, nit: please use `Number()`

Reply:
why does `inputSats` have to be a state var? seems like it's derived from `inputSatsText` only, so one could always just use `inputSatsText`? I am probably missing something.

Reply:
no you don't miss anything. I did refactor this in send a while ago, we should refactor this in receive as well.

Reply:
Refactored keeping the text version as state and converting with `Number()` when needed :+1: